frontend/pages/Homepage/cards/Software/Software.tsx

```diff
@@ -150,6 +151,10 @@ const Software = ({
       onSuccess: () => {
         setIsLoadingSoftware(false);
       },
+      // TODO: error UX?
+      onError: () => {
+        setIsLoadingSoftware(false);
```

Comment:
Good question for Noah. A more broad question is "Can we define a global treatment for failed API requests, or do we need to address them on a case-by-case basis?" 

A global way I've seen this handled is to simply throw an error notification popup saying the API request failed (in user friendly terms.) However, Noah may want to address failures on a case-by-case basis to optimize UX.